mmv1/third_party/terraform/resources/resource_dataflow_flex_template_job.go.erb

```diff
@@ -128,6 +129,13 @@ func resourceDataflowFlexTemplateJobCreate(d *schema.ResourceData, meta interfac
 	}
 
 	job := response.Job
+
+	//adding wait time for setting all the parameters into state file
+	err = waitForDataflowJobState(d, config, job.Id, userAgent, d.Timeout(schema.TimeoutUpdate), "JOB_STATE_RUNNING")
+	if err != nil {
+		return fmt.Errorf("Error waiting for job with job ID %q to be running: %s", d.Id(), err)
```

Comment:
Question: Do we already have d.Id()? Or should it be job.Id in the error message?

Reply:
Yep actually It should be a job.Id, Thanks for pointing out that.